compiler/rustc_const_eval/src/interpret/machine.rs

```diff
@@ -10,7 +10,7 @@ use rustc_middle::mir;
 use rustc_middle::ty::{self, Ty, TyCtxt};
 use rustc_span::def_id::DefId;
 use rustc_target::abi::Size;
-use rustc_target::spec::abi::Abi;
+use rustc_target::spec::abi::Abi as FnAbi;
```

Comment:
`FnAbi` is already an existing type in `rustc_target::abi::call`. This might be confusing.

Reply:
Maybe SpecAbi?

Reply:
I don't under the term "spec" here at all. For me a "spec" is a "specification", aka defining preconditions and postconditions for the purpose of formal verification...